### ISSUE TITLE
problem to installing (You don't have a valid activator configuration class)

### ISSUE THREAD

**Issue by masoudhosseiny** [nWidart/laravel-modules#1103]
thanks for your great repo!
I have this problem during installation

```
 You don't have a valid activator configuration class. This might be due to your config being out of date.                                   
   Run php artisan vendor:publish --provider="Nwidart\Modules\LaravelModulesServiceProvider" --force to publish the up to date 
configuration  
```

**Comment by nWidart** [comment]
Hi,

The error message is telling you what to do:
Run
```
php artisan vendor:publish --provider="Nwidart\Modules\LaravelModulesServiceProvider" --force
```

**Comment by masoudhosseiny** [comment]
I tried this comant but I got the same error message! 
I ran: 
`php artisan vendor:publish --provider="Nwidart\Modules\LaravelModulesServiceProvider" --force`
and I got : 
` You don't have a valid activator configuration class. This might be due to your config being out of date.                                   
   Run php artisan vendor:publish --provider="Nwidart\Modules\LaravelModulesServiceProvider" --force to publish the up to date 
configuration  `
